tests/test_vec_normalize.py

```diff
@@ -76,6 +76,42 @@ def test_vec_env(tmpdir):
     check_vec_norm_equal(norm_venv, deserialized)
 
 
+def _make_stepped_cartpole():
+    venv = DummyVecEnv([lambda: gym.make("CartPole-v1")])
+    venv = VecNormalize(venv)
+    venv.reset()
+    venv.get_original_obs()
+
+    for _ in range(100):
+        acts = [venv.action_space.sample()]
+        venv.step(acts)
+    return venv
+
+
+def test_get_original():
+    venv = _make_stepped_cartpole()
+    acts = [venv.action_space.sample()]
+    obs, rews, _, _ = venv.step(acts)
+    orig_obs = venv.get_original_obs()
+    orig_rews = venv.get_original_reward()
+    assert not np.array_equal(orig_obs, obs)
```

Comment:
Can we add some more checks here? Ideas:
  - shape and type of `orig_obs` and `obs` should match
  - does `venv.normalize_obs(orig_obs)` match `obs` and ditto for reward?

Reply:
Added these checks and also loop the normalization checks over 3 steps of CartPole now.